os/arch/arm/src/amebasmart/amebasmart_usb.c

```diff
@@ -800,4 +816,53 @@ void usb_initialize(void)
 	}
 }
 
-
+/****************************************************************************
+ * Name: usb_up_lowputc
+ *
+ * Description:
+ *   Output one packet on the serial console
+ *
+ * Input Parameters:
+ *   *ch - chatacter buffer to output
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+void usb_up_lowputc(uint8_t *ch, uint16_t len)
+{
+	uint16_t remain_data_len = len;
+	uint16_t send_data_len = 0;
+	uint8_t *buffer = NULL;
+	buffer = (uint8_t *)rtw_zmalloc(CONFIG_CDC_ACM_BULK_IN_XFER_SIZE);
```

Comment:
we can't use alloc during log level log output

Reply:
I will change it to static buffer instead